### DISCUSSION TITLE
edge browser can't fetch data.

### DISCUSSION THREAD

**Discussion by tytom2003** [reduxjs/redux-toolkit#4691] [General]
chrome, firefox can fetch new data and auto re-fetch data
edge can't  fetch new data and auto re-fetch data.
when i go to some page at first time, it will show below error. How to solve the problem? Edge browser cannot compatible redux-toolkit?
![error1](https://github.com/user-attachments/assets/dbfbc5af-ca67-4411-ba46-c2fd19c89be5)

**Comment by EskiMojo14** [comment]
the error indicates you're trying to use an object as a React node, e.g. `<div>{myObj}</div>`. This seems unrelated to RTK.
